Assets/HoloToolkit/Utilities/Scripts/ApplicationViewManager.cs

```diff
@@ -49,7 +49,8 @@ private void Start()
         {
             var viewId = ApplicationView.GetForCurrentView().Id;
             var view = CoreApplication.GetCurrentView();
-            if (CallbackDictionary.TryRemove(viewId, out var cb))
```

Comment:
The commit says these are coming from the dev branch, but I don't see these changes there. Do they need to be ported over?

Reply:
They should be there. I didn't get a chance to test out the app view manager just yet though.

Reply:
Mostly I just removed the c# language features

Reply:
But I also didn't solve the issue with IL2CPP builds:

```
Assets\MixedRealityToolkit\Utilities\Scripts\ApplicationViewManager.cs(98,36): error CS1545: Property, indexer, or event 'CoreWindow.VisibilityChanged' is not supported by the language; try directly calling accessor methods 'CoreWindow.add_VisibilityChanged(TypedEventHandler<CoreWindow, VisibilityChangedEventArgs>)' or 'CoreWindow.remove_VisibilityChanged(EventRegistrationToken)'
```

Reply:
Might just roll back this change.

Reply:
@StephenHodgson, did you decide to roll back this change?

Reply:
The changes fix the build issues, but I haven't had a chance to test them

Reply:
I would like to get this into the next release though

Reply:
next == may?

Reply:
hopefully